### DISCUSSION TITLE
Fitting endosomal clearance parameter for large molecules

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1653] [Questions & Problems]
I am currently in the process of making a large molecule drug model. However, currently it seems like the endosomal clearance of the drug is a bit too low, based on the observed data that I have. I can either increase the rate constant for endosomal uptake, or decrease the rate constant for recycling from the endosomal space. If we assume that different large molecules have different endosomal clearance, which one of the two parameters would I adjust to make it more physiologically based? Does it make more sense to say more of the drug enters the endosome, or does it make more sense to say less of it gets recycled?

**Comment by StephanSchaller** [comment]
Hi @akim2000, how does the observed data inform endosomal CL? Did you specify a Kd for FcRn binding? This Kd defines the fraction recycled. As it is a drug-specific parameter, it makes sense to adjust the Kd. But if you left it at default ( very large number) it is already at maximum.
Do you think there could be other CL pathways?

Best,
Stephan

**Reply by akim2000** [reply]
Isn't the Kd related to the binding process between the drug and the FcRn? I guess adjusting Kd can also increase or decrease CL, but when would you adjust just the Kd of the drug, instead of adjusting the actual recycling rate of the drug-FcRn complex out of the endosome, or even the endosomal uptake rate of the drug?

**Reply by StephanSchaller** [reply]
> Isn't the Kd related to the binding process between the drug and the FcRn?

Exactly, so this is drug-specific. Isn't this what you wanted? So you might have antibodies with lower or higher Kd for the binding...

Recycling rate of the drug-FcRn complex out of the endosome, or even the endosomal uptake rate of the drug are (in principle)  physiologic parameters and should not vary between drugs.

**Reply by Christoph27** [reply]
Keep also in mind that measured Kd(FcRn) values are assay dependent and a certain set of Kd(FcRn)  values were used during model development (especially for fitting endosomal FcRn concentrations). It is thus fair to adjust Kd(FcRn) if the value is not too far away from the respective value used in https://doi.org/10.1007/s10928-017-9559-4 (e.g. 0.63 µM or 0.85 µM for human/humanized Ab in human. From my experience I prefer the  0.85 µM. For human/humanized Ab in human, I start usually by setting Kd(FcRn) to 0.85 µM and fit other CL pathways e.g. target mediated CL).

**Reply by wilbertdew** [reply]
Hi @akim2000, both endosomal uptake and FcRn recycling are considered system-specific parameters (although there might be some influence of the large molecule properties), and we prefer adjusting the molecule-specific properties. This keeps the phyioslogical model constant and retains the information of all the other molecules that were described by this model with the default parameters for endosomal uptake and FcRn recycling.

**Reply by prvmalik** [reply]
@Christoph27 that approach works like a charm for me
